merkletree_test.go

```diff
@@ -0,0 +1,312 @@
+package merkletree
+
+import (
+	"bytes"
+	"crypto/rand"
+	"reflect"
+	"testing"
+
+	"github.com/coniks-sys/libmerkleprefixtree-go/internal"
+
+	"golang.org/x/crypto/ed25519"
+	"golang.org/x/crypto/sha3"
+)
+
+var treeNonce = []byte("TREE NONCE")
+var salt = []byte("salt")
+
+var (
+	pk, sk []byte
+)
+
+func init() {
+	pk, sk, _ = ed25519.GenerateKey(rand.Reader)
```

Comment:
`init` autom. get called before everything else in this file: https://golang.org/doc/effective_go.html#init
But I also think it couldn't harm to move the key-generation to specific tests instead of in an init function.